### DISCUSSION TITLE
--download-db-only and --download-java-db-only can't be used in together

### DISCUSSION THREAD

**Discussion by baskervilski** [aquasecurity/trivy#7250] [Bugs]
### Description

Version: 0.53.0

Running
`trivy image --download-db-only --download-java-db-only`
or:
`trivy image --download-java-db-only --download-db-only`

results with only the Java DB being downloaded, without any message/warning that the other DB won't be downloaded.

### Desired Behavior

EITHER: Download both DBs when both flags are present.
OR: Raise an error if both flags are used simultaneously.

### Actual Behavior

<img width="1129" alt="image" src="https://github.com/user-attachments/assets/ac1d2c78-ddb3-4934-9c56-bf1d6f839c97">

<img width="1032" alt="image" src="https://github.com/user-attachments/assets/73e65f90-776a-4b73-be16-875ab88e938e">

### Reproduction Steps

```bash
1. `trivy clean --all`
2. `trivy image --download-db-only --download-java-db-only`
3. `trivy image ubuntu:22.04 --skip-db-update --skip-java-db-update`
...
```

### Target

Container Image

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
NA
```

### Operating System

Linux

### Version

```bash
0.53.0
```

### Checklist

- [X] Run `trivy clean --all`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by knqyf263** [comment]
@baskervilski Thanks for your report.

@DmitriyLewen We should return an error for clarity if both flags are specified.

**Reply by DmitriyLewen** [reply]
Created #7259

**Comment by DmitriyLewen** [comment]
`v0.54.0` will return an error if both `--download-java-db-only` and `--download-db-only` flags are used.
